rma_sale/views/sale_portal_template.xml

```diff
@@ -32,18 +32,17 @@
                             </div>
                             <t
                                 t-set="delivery_addresses"
-                                t-value="sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
+                                t-value="sale_order.partner_shipping_id | sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
```

Comment:
A sales order always have `partner_shipping_id`:
```suggestion
                                t-value="sale_order.partner_shipping_id"
```

Reply:
This a bitwise operator not a boolean one

Reply:
And which is its purpose then?

Reply:
To have a recordset of the commercial entity children and the original shipping address (wich could be not on the other set) avoiding repeating the last

Reply:
OK